src/styles.css

```
@@ -708,3 +708,68 @@ footer a:hover{text-decoration:underline}
+   TABLET (768–992px)
+   =============================== */
+@media (min-width: 768px) and (max-width: 992px) {
+  .nav {
+    padding: 14px 18px;
+  }
+
+  .nav-links {
+    gap: 16px;
+  }
+  }
+git commit --amend --signoff
```

Stylelint (17.12.0)

[error] Unknown word git (CssSyntaxError)

(CssSyntaxError)